src/scripts/exclusion_term_expansion/exclusion_term_expansion.py

```diff
@@ -0,0 +1,299 @@
+"""Takes in the full ontology and the exclusions tsv and extracts a simple list of terms from it.
+Outputs two files:
+1. a simple list of terms (config/ONTO_NAME_term_exclusions.txt)
+2. a simple two column file with the term_id and the exclusion reason
+
+# Resources
+- GitHub issue: https://github.com/monarch-initiative/mondo-ingest/issues/22
+
+TODO's
+  - todo: Do we need to worry about edge case where exclusion table is empty? (e.g. we include such a table for all
+     ontologies, but there exists 1 where we don't exclude anything.
+  - todo: later: see below: #x1: regex results: would be good to filter out by IRI or prefix that is not from source
+     ontology, but how to determine the appropriate prefix just based on the 2 file params, but could have edge cases.
+  - todo: later: see below: #x2
+  - todo: later: see below: #x3
+  - todo: later: see below: #x4: Switch from SPARQL to OAK.
+  - todo: later: When constructing exclusion TSV: bug: ICD10CM:C7A-C7A reported as term, but should be ICD10CM:C7A
```

Comment:
This is actually a bug I'll need to address in the ICD10CM expansion work done in `mondo-analysis`: https://github.com/monarch-initiative/mondo-analysis/issues/37

Reply:
Was this fixed?

Reply:
This one was a rare exception where this was actually a legit term: https://www.icd10data.com/ICD10CM/Codes/C00-D49/C7A-C7A

Removing the todo